descope/auth/sdk.go

```diff
@@ -248,25 +220,19 @@ type Authentication interface {
 	// specified roles.
 	// This is a shortcut for ValidateTenantRoles(token, "", roles)
 	ValidateRoles(token *Token, roles []string) bool
+
 	// ValidateTenantRoles - Use to ensure that a validated session token has been granted
 	// the specified roles for a specific tenant.
 	ValidateTenantRoles(token *Token, tenant string, roles []string) bool
 
 	// DeleteCookies - Deletes the session and refresh cookies in the http response.
 	// Use the ResponseWriter (optional) to apply the cookies to the response automatically.
-	// This is a shortcut for DeleteCookiesWithOptions(r, WithResponseOption(w))
 	DeleteCookies(request *http.Request, w http.ResponseWriter) error
-	// DeleteCookiesWithOptions - Deletes the session and refresh cookies in the http response.
-	DeleteCookiesWithOptions(request *http.Request, options ...Option) error
 
 	// Logout - Use to perform logout from all active sessions for the request user. This will revoke the given tokens
 	// and if given options will also remove existing session on the given response sent to the client.
```

Comment:
we may need to go over the docs again and make sure no options leftovers are still documented @slavikm @aviadl